### DISCUSSION TITLE
API C# GetIssuedDocumentResponse

### DISCUSSION THREAD

**Discussion by Mustino2022** [fattureincloud/api#148] [Q&A] [answered]
Buongiorno,
nella classe GetIssuedDocumentResponse  è disponibile una proprietà per la data di creazione e la data di aggiornamento?

![image](https://user-images.githubusercontent.com/117667863/200529520-21671801-d876-4a7e-916e-83f44a21d5bb.png)

**Comment by valmoz** [comment, marked as answer]
@Mustino2022 buongiorno, verrà rilasciata nelle prossime versioni della SDK.

**Reply by Mustino2022** [reply]
Buongiorno @valmoz attenderò la nuova versione.

**Reply by emajo** [reply]
Ciao @Mustino2022 abbiamo rilasciato tutte le sdk con questa modifica!
